src/installing-workflow/installing-deis-workflow.md

```diff
@@ -43,9 +43,11 @@ $ helm repo add deis https://github.com/deis/charts
 Now that you have Helm installed and have added the Deis Chart Repository, install Workflow by running:
 
 ```
-$ helm fetch deis/deis                 # fetches the chart into a local workspace
-$ helm generate -x manifests deis      # generates various secrets
-$ helm install deis                    # injects resources into your cluster
+$ helm fetch deis/workflow             # fetches the chart into a local
+                                       # workspace
```

Comment:
Note the new manual line break in the comment and continuation on the next line has everything to do with the width of the code windows in the formatted docs.